SECURITY.md

```diff
@@ -0,0 +1,60 @@
+# Security Process and Policy
+
+This document provides the details on the Veriason security policy and details the processes
```

Comment:
> The one comment I have is about scope: since this is supposed to apply globally, I think we should create a `github.com/veraison/policies` repo and move this (and similarly scoped) content there. Then have each repo link the relevant bits.

Adding the link to the [community repo](https://github.com/veraison/community) we created. 
To avoid losing the feedback, should we complete this PR, then move the file to the community repo?